vowpalwabbit/search.cc

```diff
@@ -2555,6 +2557,8 @@ base_learner* setup(VW::setup_base_i& stack_builder)
   new_options.add(make_option("search_task", task_string)
                       .keep()
                       .necessary()
+                      .one_of({"sequence", "sequencespan", "sequence_ctg", "argmax", "sequence_demoldf",
```

Comment:
Based on the help string shouldnt "list" be one of these?

Reply:
yes good catch!